config/packages/content_negotiation.yaml

```diff
@@ -0,0 +1,10 @@
+content_negotiation:
+    formats:
+        - path: /ping$
+          priorities:
+        - path: ^/
+          priorities: xml
+    locales:
+        - path: ^/
+          priorities: '%env(POSSIBLE_LOCALES)%'
+          optional: true
```

Comment:
Means that error messages can be translated. Content is unaffected, and done at the route-level later on.